src/index.js

```diff
@@ -13,3 +13,10 @@ export { default as LayoutContainer } from "./components/LayoutContainer/LayoutC
  * buttons
  */
 export { ActionButton } from "./components/ActionButton/ActionButton";
+
+/**
+ * Changes to this component need to be made in future commits
+ */
+// export { ReportAProblem } from "./components/ReportAProblem/ReportAProblem"
+
+export { Footer } from "./components/Footer/Footer";
```

Comment:
since ReportAProblem and Footer components are still in development can we remove them from this file? I will write a note on this file but we should only include components here that are ready to be released.